### PR TITLE
[CDAP-8198] [CDAP-8199] Dataset and Stream Tracker iframe

### PR DESCRIPTION
**Pull request by elfenheart**
JIRAs:
- https://issues.cask.co/browse/CDAP-8198
- https://issues.cask.co/browse/CDAP-8199

### CODE REVIEW

File: cdap-ui/app/tracker/controllers/tracker-enable-ctrl.js [hunk 1]
```diff
@@ -169,6 +169,12 @@ class TrackerEnableController{
       return;
     }
 
+    if (this.$state.params.sourceUrl) {
+      window.location.href = decodeURIComponent(this.$state.params.sourceUrl);
+    } else {
+      this.$state.go('tracker.home');
+    }
+
     this.$state.go('tracker.home');
```
Comment (tonybach) [inline cdap-ui/app/tracker/controllers/tracker-enable-ctrl.js]: Unnecessary now with the conditional added? Or maybe keep this and remove the `else` part.

**Comment by ajainarayanan** [comment]
🎉 🎆 Awesome! Works seamlessly. Just noticed the header flashing before rendering the content. I think you are working on that separately I guess @elfenheart ?

Except for the comment from @tonybach LGTM 👍

**Comment by elfenheart** [comment]
@ajainarayanan yeah will have to fix that header bar changes after the new global nav is merged

**Comment by elfenheart** [comment]
builds: http://builds.cask.co/browse/CDAP-DRC5529-2

**Comment by ajainarayanan** [comment]
👍 LGTM

**Pull request merged**